src/analyzer/protocol/rawstream/README

```diff
@@ -0,0 +1,4 @@
+# TCP application analyzer for handing raw stream data to script-land.
+#
+# This is similar to tcp_contents(), but can be triggered via DPD,
+# well-known-port registration, or HTTP and WebSocket upgrade mechansisms
```

Comment:
I'd start this more directly in terms of what the analyzer does, not with a comparison to `tcp_contents()`. Then maybe afterwards do the compare to `tcp_contents()`.